framework/src/org/checkerframework/framework/test/TestUtilities.java

```diff
@@ -57,6 +57,39 @@
         return getJavaFilesAsArgumentList(dirs);
     }
 
+    public static List<List<File>> findJavaFilesPerDirectory(File parent, String... dirNames) {
```

Comment:
A line or two of Javadoc for this new method and the one immediately below would be helpful.